skyportal/handlers/api/internal/plot.py

```diff
@@ -184,3 +184,20 @@ def get(self, obj_id, telescope_id):
 
         self.verify_and_commit()
         return self.success(data=json)
+
+
+class FilterWavelengthHandler(BaseHandler):
+    @auth_or_token
+    def post(self):
+        data = self.get_json()
+        filters = data.pop("filters", None)
+        if filters:
+            wavelengths = []
+            for filter in filters:
+                try:
+                    # divide by 10 to convert angstrom -> nm
+                    wavelengths.append(plot.get_effective_wavelength(filter) / 10)
```

Comment:
We should make a decision to always use angstrom (which is presumably what you get from `sncosmo`) OR nanometers (which is what you get by dividing angstrom by 10). 
If we decide everything is in nm then the divide by 10 should just happen in `get_effective_wavelength` (in that case, you'd have to change the `get_color` function in `skyportal/plot.py`. If we want to keep using angstrom everywhere then this function should not divide by 10, and only the vega plot should be modified to handle angstrom instead of nm.